src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.task.TaskList;
+
+/**
+ * Represents the command for asking Duke to list its tasks in the TaskList.
+ */
+public class ListCommand implements Command{
```

Comment:
Should there be a whitespace before the open curly braces?

```suggestion
public class ListCommand implements Command {
```